.github/workflows/sanity-checks.yml

```diff
@@ -0,0 +1,58 @@
+on:
+  pull_request:
+name: Sanity checks
+concurrency:
+  group: 'tests-${{ github.event.pull_request.head.label || github.head_ref || github.ref }}'
+  cancel-in-progress: true
+permissions:
+    id-token: write
+    contents: read
+jobs:
+  test-n-lint:
+    name: ${{ matrix.target }}
+    runs-on: ubuntu-latest-8-cores
+    strategy:
+      matrix:
+        target: [test-general, test-integration,
+                 lint-fmt, lint-clippy, cargo-build] # ,lint-taplo]
```

Comment:
Should be `list-taple` reenable?

Reply:
Our intention was to re-enable it in a follow up PR in case it doesnt work out of the box. WDYT?

Reply:
Every time I enabled it it failed consistently, I think it requires a bit of work, I agree on dealing with it in separate PRs